server/php-engine.cpp

```diff
@@ -2071,6 +2107,11 @@ void parse_main_args(int argc, char *argv[]) {
   parse_option("job-workers-shared-memory-size", required_argument, 2017, "the total size of shared memory used for job workers related communication");
   parse_option("job-workers-shared-messages", required_argument, 2018, "the total count of the shared messages for job workers related communication");
   parse_option("lease-stop-ready-timeout", required_argument, 2019, "timeout for RPC_STOP_READY acknowledgement waiting in seconds (default: 0)");
+  parse_option("mysql-user", required_argument, 2020, "MySQL user");
+  parse_option("mysql-password", required_argument, 2021, "MySQL password");
+  parse_option("mysql-host", required_argument, 2022, "MySQL host");
```

Comment:
I think it's a bit strange to have customizable mysql host and not to have option for setting mysql port. What do you think about this?

Reply:
I agree with you, I just forgot about port customization, updated it, thanks

Reply:
Oh, sorry for that, It turns out such an option already exists. It's `-Q` option. So new option isn't needed actually :(

Reply:
Kek, exacltly) No problems, fixed